### ISSUE TITLE
Nats-server as Windows Service 1067 The Process Terminated Unexpectedly

### ISSUE THREAD

**Issue by mrbastian** [nats-io/nats-server#1317]
We have a client environment where NATS version 1.4.1 on Windows Server 2016 Datacenter was running for over a year with no issues on both a test server and a separate production server. Last year the service began reporting the above error on the TEST server when starting and despite all the troubleshooting we could think of, we were never able to get the service running again. It still worked on the production server until recently when it started getting the same error. We can now only run it from the command line in a logged in desktop account that we have to be sure to keep active which is far from ideal.

The service is still working with no issues whatsoever in all of our other client environments and we've run out of ideas for this one. Hoping somebody has some ideas and recommendations for what we can do to troubleshoot this further

Additional notes:

- The service is configured to run as the local system account with parameter -c c:/..../path-to-config-file.
- Also tried running as other admin users, same results.
- We've also tried installing with all config options in the SC service command line (same results), and with no parameters at all (same results).
- Running from the command line (not as a service) with same parameters works perfectly.
- Installing & running latest version (2.1.4) behaves exactly the same (1067 error as windows service on startup, works fine from command line)
- With logging enabled (debug: true, trace: true), when the service is started the log file will be created if it doesn't exist, but nothing is written to the log before the error dialog is displayed.
- Also tried configuring with NSSM to execute nats in command line mode within a service: nssm starts, but gnatsd still doesn't run

**Comment by mrbastian** [comment]
Update - apparently fixed

Based on [this page ](https://www.drivereasy.com/knowledge/solved-error-1067-process-terminated-unexpectedly-windows-10-7-8/), we tried running the following from an admin command prompt which found and "fixed" some issues:

sfc /scannow

The only thing interesting in c:\windows\logs\cbs\cbs.log was "CSI meta data corruption found".

After that we also ran the following two commands, neither of which reported any issues
DISM /Online /Cleanup-Image /CheckHealth
DISM /Online /Cleanup-Image /ScanHealth

Finally we rebooted, and the Nats-server (v2.1.4) which was configured to start automatically... WAS RUNNING!!

Sorry to bother anyone. I'll mark the issue closed.
